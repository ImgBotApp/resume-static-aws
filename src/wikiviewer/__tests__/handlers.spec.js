/* @flow */
'use strict';

test.only('', () => {});
