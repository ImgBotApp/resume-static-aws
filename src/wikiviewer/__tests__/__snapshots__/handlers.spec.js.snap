// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WikiViewer Handlers keyPressHandler should call refreshResults on 'Enter' key 1`] = `"<body><meta charset=\\"utf-8\\"><meta http-equiv=\\"x-ua-compatible\\" content=\\"ie=edge\\"><meta name=\\"description\\" content=\\"\\"><meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, height=device-height\\"><h1></h1><div class=\\"view\\"></div><div class=\\"center-panel\\"><input id=\\"search-text\\"><button id=\\"search-btn\\"></button><button id=\\"random-btn\\"></button></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Big Boss</h1><h2></h2></div><p>Big Boss disambiguation page.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Bigg Boss</h1><h2></h2></div><p>Bigg Boss is a television reality show broadcast on Colors channel in India.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss</h1><h2></h2></div><p>A boss is a person in charge, or a supervisor.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss (architecture)</h1><h2></h2></div><p>In architecture, a boss is a knob or protrusion of stone or wood.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss (crime)</h1><h2></h2></div><p>A crime boss, crime lord, mob boss, kingpin, criminal mastermind, or Don is a person in charge of a criminal organization.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss (video gaming)</h1><h2></h2></div><p>In video gaming, a boss is a significant computer-controlled enemy.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss Corporation</h1><h2></h2></div><p>Boss is a manufacturer of effects pedals for electric guitar and bass guitar.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Hugo Boss</h1><h2></h2></div><p>Hugo Boss AG, often styled as BOSS, is a German luxury fashion house.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Political boss</h1><h2></h2></div><p>A boss, in politics, is a person who controls a unit of a political party, although he/she may not hold political office.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Undercover Boss</h1><h2></h2></div><p>Undercover Boss is a television franchise series created by Stephen Lambert and produced in many countries.</p></div></body>"`;

exports[`WikiViewer Handlers refreshResults should call updateDOM with results of search 1`] = `"<body><meta charset=\\"utf-8\\"><meta http-equiv=\\"x-ua-compatible\\" content=\\"ie=edge\\"><meta name=\\"description\\" content=\\"\\"><meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, height=device-height\\"><h1></h1><div class=\\"view\\"></div><div class=\\"center-panel\\"><input id=\\"search-text\\"><button id=\\"search-btn\\"></button><button id=\\"random-btn\\"></button></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Big Boss</h1><h2></h2></div><p>Big Boss disambiguation page.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Bigg Boss</h1><h2></h2></div><p>Bigg Boss is a television reality show broadcast on Colors channel in India.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss</h1><h2></h2></div><p>A boss is a person in charge, or a supervisor.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss (architecture)</h1><h2></h2></div><p>In architecture, a boss is a knob or protrusion of stone or wood.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss (crime)</h1><h2></h2></div><p>A crime boss, crime lord, mob boss, kingpin, criminal mastermind, or Don is a person in charge of a criminal organization.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss (video gaming)</h1><h2></h2></div><p>In video gaming, a boss is a significant computer-controlled enemy.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss Corporation</h1><h2></h2></div><p>Boss is a manufacturer of effects pedals for electric guitar and bass guitar.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Hugo Boss</h1><h2></h2></div><p>Hugo Boss AG, often styled as BOSS, is a German luxury fashion house.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Political boss</h1><h2></h2></div><p>A boss, in politics, is a person who controls a unit of a political party, although he/she may not hold political office.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Undercover Boss</h1><h2></h2></div><p>Undercover Boss is a television franchise series created by Stephen Lambert and produced in many countries.</p></div></body>"`;

exports[`WikiViewer Handlers searchHandler should call refreshResults and updateDOM 1`] = `"<body><meta charset=\\"utf-8\\"><meta http-equiv=\\"x-ua-compatible\\" content=\\"ie=edge\\"><meta name=\\"description\\" content=\\"\\"><meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, height=device-height\\"><h1></h1><div class=\\"view\\"></div><div class=\\"center-panel\\"><input id=\\"search-text\\"><button id=\\"search-btn\\"></button><button id=\\"random-btn\\"></button></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Big Boss</h1><h2></h2></div><p>Big Boss disambiguation page.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Bigg Boss</h1><h2></h2></div><p>Bigg Boss is a television reality show broadcast on Colors channel in India.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss</h1><h2></h2></div><p>A boss is a person in charge, or a supervisor.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss (architecture)</h1><h2></h2></div><p>In architecture, a boss is a knob or protrusion of stone or wood.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss (crime)</h1><h2></h2></div><p>A crime boss, crime lord, mob boss, kingpin, criminal mastermind, or Don is a person in charge of a criminal organization.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss (video gaming)</h1><h2></h2></div><p>In video gaming, a boss is a significant computer-controlled enemy.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss Corporation</h1><h2></h2></div><p>Boss is a manufacturer of effects pedals for electric guitar and bass guitar.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Hugo Boss</h1><h2></h2></div><p>Hugo Boss AG, often styled as BOSS, is a German luxury fashion house.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Political boss</h1><h2></h2></div><p>A boss, in politics, is a person who controls a unit of a political party, although he/she may not hold political office.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Undercover Boss</h1><h2></h2></div><p>Undercover Boss is a television franchise series created by Stephen Lambert and produced in many countries.</p></div></body>"`;

exports[`WikiViewer Handlers updateDOM should populate given elements with returned data 1`] = `"<body><meta charset=\\"utf-8\\"><meta http-equiv=\\"x-ua-compatible\\" content=\\"ie=edge\\"><meta name=\\"description\\" content=\\"\\"><meta name=\\"viewport\\" content=\\"width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no, height=device-height\\"><h1></h1><div class=\\"view\\"></div><div class=\\"center-panel\\"><input id=\\"search-text\\"><button id=\\"search-btn\\"></button><button id=\\"random-btn\\"></button></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Big Boss</h1><h2></h2></div><p>Big Boss disambiguation page.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Bigg Boss</h1><h2></h2></div><p>Bigg Boss is a television reality show broadcast on Colors channel in India.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss</h1><h2></h2></div><p>A boss is a person in charge, or a supervisor.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss (architecture)</h1><h2></h2></div><p>In architecture, a boss is a knob or protrusion of stone or wood.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss (crime)</h1><h2></h2></div><p>A crime boss, crime lord, mob boss, kingpin, criminal mastermind, or Don is a person in charge of a criminal organization.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss (video gaming)</h1><h2></h2></div><p>In video gaming, a boss is a significant computer-controlled enemy.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Boss Corporation</h1><h2></h2></div><p>Boss is a manufacturer of effects pedals for electric guitar and bass guitar.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Hugo Boss</h1><h2></h2></div><p>Hugo Boss AG, often styled as BOSS, is a German luxury fashion house.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Political boss</h1><h2></h2></div><p>A boss, in politics, is a person who controls a unit of a political party, although he/she may not hold political office.</p></div><div class=\\"result-container\\"><div class=\\"heading\\"><h1>Undercover Boss</h1><h2></h2></div><p>Undercover Boss is a television franchise series created by Stephen Lambert and produced in many countries.</p></div></body>"`;
